nixos/modules/security/sudo.nix

```
@@ -30,61 +32,67 @@ in
 
   ###### interface
 
-  options = {
+  options.security.sudo = {
+
+    defaultOptions = mkOption {
+      type = with types; listOf str;
+      default = [ "SETENV" ];
+      description = mdDoc ''
+        Options used for the default rules, granting `root` and the
+        `wheel` group permission to run any command as any user.
+      '';
+    };
 
-    security.sudo.enable = mkOption {
+    enable = mkOption {
       type = types.bool;
       default = true;
```

GitHub doesn't let me suggest accross this diff, but this goes for the whole option

```suggestion
    enable = lib.mkEnableOption (mdDoc
      "the {command}`sudo` command, which allows non-root users to execute commands as root"
    );
```